src/ansys/grantami/bomanalytics/_typing.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, TypeVar, Union
+
+from ansys.openapi.common import Unset_Type
+
+Any_Type = TypeVar("Any_Type", bound=Any)
+
+
+def _cast_unset_union_to_any(value: Union[Any_Type, Unset_Type]) -> Union[Any_Type]:
```

Comment:
Just trying to see if I'm getting this right. 

`Any_Type` is required to indicate that the returned value is of the same type as the input value (if it wasn't Unset)? Sort of a Generic situation?

Can the return be `Any_Type` or is the Union actually required?

Reply:
Ah, Union is not needed. As you say, it's a generic function as you say, the Any_Type typevar just makes it function for any type.

I'll fix the union